src/style.css

```
@@ -1,3 +1,131 @@
+/* reset style */
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+ul,
+ol {
+  padding: 0;
+}
+
+h1,
+h2,
+h3,
+h4,
+p,
+ul[class],
+ol[class],
+li,
+figure,
+figcaption,
+blockquote,
+dl,
+dd {
+  margin: 0;
+}
+
 body {
+  min-height: 100vh;
+  scroll-behavior: smooth;
+  text-rendering: optimizeSpeed;
+  line-height: 1.5;
+  font-family: "Roboto", sans-serif;
   margin: 0;
+  font-weight: 700;
+  font-size: 12px;
+}
+
+ul[class],
+ol[class] {
+  list-style: none;
+}
+
+a:not([class]) {
+  text-decoration-skip-ink: auto;
+}
+
+img {
+  max-width: 100%;
+  display: block;
+}
+
+input,
+button,
+textarea,
+select {
+  font: inherit;
+}
+
+@media (prefers-reduced-motion: reduce) {
+  * {
+    animation-duration: 0.01ms !important;
+    animation-iteration-count: 1 !important;
+    transition-duration: 0.01ms !important;
+    scroll-behavior: auto !important;
+  }
+}
+/* main style header navigation */
+
+.header {
+  display: flex;
+  align-items: center;
+  margin: 10px 50px;
+}
+
+.navbar {
+  display: flex;
+}
+
+.navlist {
+  margin-left: 20px;
+}
+
+.navlink {
+  text-decoration: none;
+  color: black;
+  position: relative;
+  padding-top: 15px;
+  padding-bottom: 15px;
```

links should have a bigger padding value
```suggestion
  padding: 23px 0;
```